### PR TITLE
fix chart: Values.multus.multusCNI.uninstall does not take effect

### PR DESCRIPTION
**Pull request by cyclinder**
#### What type of PR is this?
- kind/bug

**What this PR does / why we need it**:

chart: make Values.multus.multusCNI.uninstall to sense

**Which issue(s) this PR fixes**:

Fixes https://github.com/spidernet-io/spiderpool/issues/2972

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by weizhoublue** [comment]
sign off

File: charts/spiderpool/templates/daemonset.yaml [hunk 1]
```diff
@@ -213,6 +214,7 @@ spec:
                 rm -rf /host/etc/cni/net.d/multus.d || true
                 rm -f /host/etc/cni/net.d/00-multus.conf || true
                 {{ .Values.spiderpoolAgent.binName }} shutdown
```
Comment (Icarus9913) [inline charts/spiderpool/templates/daemonset.yaml:216]: 你这直接给preStop套个if判断，这里原本的 `{{ .Values.spiderpoolAgent.binName }} shutdown` 不也被包含了？

**Comment by cyclinder** [comment]
```
##[debug]}
Redirecting to blob download url: https://productionresultssa6.blob.core.windows.net/actions-results/383bc6db-35f9-4f62-9c96-77aac19d6434/workflow-job-run-47eebfc0-3fc8-55d8-2ea4-4641d9c01654/artifacts/image-tar-spiderpool-controller.zip
Starting download of artifact to: /home/runner/work/spiderpool/spiderpool/test/.download
Error: Unable to download artifact(s): Unable to download and extract artifact: Request timeout: /actions-results/383bc6db-35f9-4f62-9c96-77aac19d6434/workflow-job-run-47eebfc0-3fc8-55d8-2ea4-4641d9c01654/artifacts/image-tar-spiderpool-controller.zip?se=2023-12-22T05%3A19%3A39Z&sig=ScLOKj8bmOzcN0rgvx%2Bq8OlWhmhoG0eI21NkdeEYoao%3D&sp=r&spr=https&sr=b&st=2023-12-22T05%3A09%3A39Z&sv=2021-12-02
```

**Review by Icarus9913 — APPROVED** [review]

**Review by weizhoublue — APPROVED** [review]

File: charts/spiderpool/templates/daemonset.yaml [hunk 2]
```diff
@@ -253,6 +255,10 @@ spec:
           mountPath: /host{{ .Values.global.cniBinHostPath }}
         - name: ipam-unix-socket-dir
           mountPath: {{ dir .Values.global.ipamUNIXSocketHostPath }}
+        {{- if .Values.multus.multusCNI.uninstall }}
```
Comment (weizhoublue) [inline charts/spiderpool/templates/daemonset.yaml:258]: ? 这个目录的 挂载，当初是因为 multus 而 加的么 ？

**Pull request merged**